app/config.py

```diff
@@ -6,3 +6,5 @@
 RATE_LIMIT = "10000/minute"
 
 BASE_PATH = os.getenv("BASE_PATH", "")
+
+DOCTYPE_URL = "https://raw.githubusercontent.com/ambanum/OpenTermsArchive/master/src/app/types.json"
```

Comment:
I was more expecting that this would come from the dataset itself, do you have this information inside ?

Reply:
Nope, there is no way to generate this config info from the dataset